django_refugeerights/django_refugeerights/settings.py

```diff
@@ -19,7 +19,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = ''
+SECRET_KEY = 'ytreureuyryru'
```

Comment:
We probably want to set this back to `''`, otherwise we won't get errors if we forget to set the secret key in production. To run it locally, you can create something like a `local_settings.py` file, and override the `SECRET_KEY` in there, and set the `local_settings.py` as the settings file to use when you run `runserver`.